rotkehlchen/assets/asset.py

```diff
@@ -85,6 +85,8 @@ def deserialize_from_db(cls, entry: UnderlyingTokenDBTuple) -> 'UnderlyingToken'
     'COIN-2': 'COIN',
     strethaddress_to_identifier('0x15B543e986b8c34074DFc9901136d9355a537e7E'): 'STCCOIN',
     strethaddress_to_identifier('0x8f136Cc8bEf1fEA4A7b71aa2301ff1A52F084384'): 'STC',
+    'BCH': 'BCHN',
+    'MER-2': 'MER',
```

Comment:
Hmmm is this supposed to solve https://github.com/rotki/rotki/issues/3361

That was supposed to be FTX, not bittrex. Or is bittrex also targetting the wrong MER? (could be -- would need to check their markets)

Reply:
As far as I know is not the case

Reply:
Yes but you probably still need to map it correctly since we now have 2 assets with symbol MER, right?

And indeed in Bittrex MER is mercury: https://global.bittrex.com/Market/Index?MarketName=BTC-MER

Reply:
Sorry. Should be correct now

Reply:
No it's not. Come on man, I wrote above in Bittrex MER is mercury. For us MER is Mercury and MER-2 Mercurial. So you need to:

```suggestion
    'MER': 'MER',
```

Reply:
Sorry, cunfused the names :clown_face: